src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -2400,8 +2415,9 @@ bool primitive_inst::is_valid_fusion() const {
         // TODO: Only fc_bf_tiled_kernel & ref kernel are verified for fused eltwise. To support more fc kernels for eltwise fusion
         if (!_node->get_selected_impl())
             return false;
-        if ((_node->get_selected_impl()->get_kernel_name().find("fully_connected_gpu_bf_tiled") == std::string::npos)
-            && (_node->get_selected_impl()->get_kernel_name().find("fully_connected_gpu_bfyx_ref") == std::string::npos)) {
+        if ((_node->get_selected_impl()->get_kernel_name().find("fully_connected_gpu_bf_tiled") == std::string::npos) &&
+            (_node->get_selected_impl()->get_kernel_name().find("fully_connected_gpu_bfyx_ref") == std::string::npos) &&
+            (_node->get_selected_impl()->get_kernel_name().find("fully_connected_gpu_imad") == std::string::npos)) {
```

Comment:
`        if ((_node->get_selected_impl()->get_kernel_name().find("fully_connected_gpu_bf_tiled") == std::string::npos) &&
            (_node->get_selected_impl()->get_kernel_name().find("fully_connected_gpu_bfyx_ref") == std::string::npos) &&
            (_node->get_selected_impl()->get_kernel_name().find("fully_connected_gpu_imad") == std::string::npos)) {`

Reply:
for i8 u8 we can allow always, could you please update?